### PR TITLE
feat: Setup the Gitpod ready-to-code system

### PR DESCRIPTION
**Pull request by Panquesito7**
### Describe the changes you've made

- Setup the Gitpod ready-to-code system.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.
 
## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/web3community/web3community.github.io/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/web3community/web3community.github.io/pull/122"></a>

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: npm
+    init: npm install
+    command: npm run build && npm run start
```
Comment (krishguptadev) [inline .gitpod.yml:5]:
```suggestion
    command: npm run start
```

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  prebuilds:
+    addBadge: true
+    addComment: false
+    addCheck: false
+    master: true
+    branches: true
+    pullRequestsFromForks: true
+
+vscode:
+  extensions:
```
Comment (krishguptadev) [inline .gitpod.yml:22]: Can we also add the astro ext here?

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (shuklaritvik06) [reply .gitpod.yml:5]: +1

File: .gitpod.yml (hunk 2 above)
Comment (shuklaritvik06) [reply .gitpod.yml:22]: +1

Comment (Panquesito7) [reply .gitpod.yml:22]: > Can we also add the astro ext here?

For sure. I'll add it. 🙂

Comment (Panquesito7) [reply .gitpod.yml:22]: Can we also add the astro ext here?

> I didn't find any. 😅

Comment (shuklaritvik06) [reply .gitpod.yml:22]: astro-build.astro-vscode

Comment (shuklaritvik06) [reply .gitpod.yml:22]: Search this I think krish is talking about this one

Comment (Panquesito7) [reply .gitpod.yml:22]: > Search this I think krish is talking about this one

Searched, nothing.

image

Comment (shuklaritvik06) [reply .gitpod.yml:22]: SCREEB


But it's showing for me 🤔

Comment (krishguptadev) [reply .gitpod.yml:22]: It also shows for me? 😅 Maybe we can add the id for now and see what happens :)

Comment (Panquesito7) [reply .gitpod.yml:22]: > It also shows for me? 😅 Maybe we can add the id for now and see what happens :)

Gitpod won't load it if it doesn't find the extension. Anyways, let's try it out. 😄

Comment (krishguptadev) [reply .gitpod.yml:22]: Oh :0, I think this is a issue at the gitpod side 🤔

File: README.md [hunk 3]
```diff
@@ -14,45 +14,47 @@ We are currently implementing the design from Figma to code. If you want to help
...
+### 5\. Build the project to generate the first CSS
 
 This step is only required the very first time.
 
 ```
 npm run build
 ```
 
-### 5\. Run Eleventy
+### 6\. Run Eleventy
```
Comment (krishguptadev) [inline README.md:45]: If this is 6 how is the next step 8

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: .gitpod.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    onOpen: open-preview
+
+github:
+  prebuilds:
+    addBadge: true
+    addComment: false
+    addCheck: false
+    master: true
+    branches: true
+    pullRequestsFromForks: true
```
Comment (krishguptadev) [inline .gitpod.yml:19]: Pull req as well?

File: .gitpod.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+# List the ports to expose. Learn more https://www.gitpod.io/docs/config-ports/
+ports:
+  - port: 3000
+    onOpen: open-preview
+
+github:
+  prebuilds:
+    addBadge: true
+    addComment: false
```
Comment (krishguptadev) [inline .gitpod.yml:15]: Doesn't this req @vinzvinci to add the app to our org?

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: .gitpod.yml (hunk 5 above)
Comment (Panquesito7) [reply .gitpod.yml:15]: > Doesn't this req @vinzvinci to add the app to our org?

Nope, plus it's already added, AFAIK.

File: .gitpod.yml (hunk 4 above)
Comment (Panquesito7) [reply .gitpod.yml:19]: > Pull req as well?

Yeah, you can see the badge in this PR. 🙂

File: .gitpod.yml (hunk 5 above)
Comment (shuklaritvik06) [reply .gitpod.yml:15]: Oh if it's added then good, otherwise for prebuild I think we have to add the app to our org

**Review by krishguptadev — APPROVED** [review]

**Pull request merged**